src/libcore/mem/maybe_uninit.rs

```diff
@@ -230,6 +231,13 @@ impl<T: Copy> Clone for MaybeUninit<T> {
     }
 }
 
+#[unstable(feature = "maybe_uninit_debug", issue = "0")]
+impl<T> fmt::Debug for MaybeUninit<T> {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        f.pad("MaybeUninit")
```

Comment:
Might as well use https://doc.rust-lang.org/nightly/std/any/fn.type_name.html here to be more informative.

Reply:
Good idea!